00-open-a-shapefile.Rmd

```diff
@@ -8,9 +8,9 @@ contributors: [Sarah Newman]
 dateCreated:  2015-10-23
 lastModified: `r format(Sys.time(), "%Y-%m-%d")`
 packagesLibraries: [rgdal, raster]
-category: [self-paced-tutorial]
+category: 
```

Comment:
Looking at the files, all that you need to do on each Rmd page is change:

category: self-paced-tutorial

To :

categories: [self-paced-tutorial]

Everything else is ok on the files. When I rebuild I'll test against the new template to make sure everything works. We can't test now on the old template files.